source/quick-reference.txt

```diff
@@ -434,16 +434,16 @@ their related reference and API documentation.
                  context.TODO(), 
                  bson.D{},
                  options.Find().SetProjection(
-                     bson.D{{"vendor", 0}, {"_id",0}}
+                     bson.D{{"point", 0}, {"_id",0}}
```

Comment:
I: the projection sets `point` to 0 but the output still includes the field
S: omit the unprojected field from the output